suites/pacific/cephadm/cephadm.yaml

```diff
@@ -22,6 +22,11 @@ tests:
           initial-dashboard-user: admin123
           initial-dashboard-password: admin123
           fsid: f64f341c-655d-11eb-8778-fa163e914bcc
+        check_cluster_state:                    # to view ceph status only
+          - "ceph status"
```

Comment:
Instead of `check_cluster_state`, should be change it to `get_cluster_details`? If they are the set of standard commands that we would be executing... would it be a good idea to support `bool`?

We can support both `bool` & `list` if it makes sense.

Reply:
I think it would good if we keep it list of CLI commands, because we may need mapping in case of Boolean and conditional check at execution.

Reply:
`get_cluster_details` looks good.